public/workers/optimizer.js

```diff
@@ -1151,10 +1155,19 @@ function optimizeMods(availableMods, characters, order, globalSettings, previous
     .concat(globalSettings.lockUnselectedCharacters ? unselectedCharacters : []);
 
   // For each not-locked character in the list, find the best mod set for that character
+  let breakForStopAt = false;
   const optimizerResults = order.reduce((modSuggestions, { id: characterID, target }, index) => {
```

Comment:
When `stopAt` is the last index rather than a character ID, this can be greatly simplified. You don't even need to have a variable for whether to break anymore, you could just do something like:

```suggestion
  const optimizerResults = order.slice(0, limit).reduce((modSuggestions, { id: characterID, target }, index) => {
```

Reply:
I have simplified this, not quite this way, because I still want to push null suggestedMods for all the characters after the selected one, but I am using an index now so it works for multiple occurrences of the same character.